pkg/api/org.go

```diff
@@ -151,7 +151,23 @@ func updateOrgAddressHelper(form dtos.UpdateOrgAddressForm, orgID int64) respons
 }
 
 // GET /api/orgs/:orgId
-func DeleteOrgByID(c *models.ReqContext) response.Response {
+func (hs *HTTPServer) DeleteOrgByID(c *models.ReqContext) response.Response {
```

Comment:
Checking if `c.OrgId` equals `c.ParamsInt64(":orgId")` should be enough, right?

Reply:
yes, this is much more simpler